### ISSUE TITLE
[feature request] support email notification

### ISSUE THREAD

**Issue by ralf0131** [pouchcontainer/pouchrobot#89]
### Ⅰ. Issue Description

For example, support send the weekly report to mailing list.

**Comment by Frankzhaopku** [comment]
Yes, all kinds of notification could be very useful to dev group, such as dingTalk or wechat notification is also very important. We will add these features.
